src/components/calcite-input/calcite-input.scss

```diff
@@ -55,60 +55,59 @@
   & input,
   & .calcite-input-prefix,
   & .calcite-input-suffix {
-    @apply text-1 p-4;
-    height: 56px;
+    @apply text-1h p-3;
+    height: theme("height.12");
   }
   & textarea {
-    min-height: 56px;
+    min-height: theme("height.12");
   }
   & .calcite-input-number-button-wrapper,
   & .calcite-input-action-wrapper calcite-button,
   & .calcite-input-action-wrapper calcite-button button {
-    height: 56px;
+    height: theme("height.12");
   }
   & textarea,
   & input[type="file"] {
     height: auto;
   }
   & .calcite-input-clear-button {
-    min-height: 56px;
-    min-width: 56px;
+    min-height: theme("height.12");
+    min-width: theme("height.12");
   }
 }
 
 // disabled styles
 :host([disabled]) {
-  pointer-events: none;
+  @apply pointer-events-none;
 
   & .calcite-input-wrapper {
-    opacity: var(--calcite-ui-opacity-disabled);
-    pointer-events: none;
+    @apply opacity-disabled pointer-events-none;
   }
   & textarea,
   & input {
-    pointer-events: none;
+    @apply pointer-events-none;
   }
 }
 
 :host textarea,
 :host input {
-  display: flex;
-  position: relative;
+  @apply flex 
+    relative 
+    max-w-full 
+    max-h-full 
+    flex-1 
+    box-border 
+    font-inherit 
+    m-0 
+    bg-foreground-1 
+    text-color-1 
+    font-normal 
+    rounded-none;
   min-width: 20%;
-  max-width: 100%;
-  max-height: 100%;
-  flex: 1;
-  box-sizing: border-box;
   -webkit-appearance: none;
-  font-family: inherit;
   transition: $transition, height 0s;
   box-shadow: 0 0 0 0 transparent;
```

Comment:
for the transition?

Reply:
I'm not sure. hehe.

Do we not need it?

Reply:
just left this as is, should I remove or do something else? I'm not sure.